src/index.js

```diff
@@ -0,0 +1,195 @@
+const cityUpdate = () => {
+  const newCityText = state.cityTextField.value;
+  state.headerCity.textContent = newCityText;
+};
+
+const cityDefault = "Boston";
+
+const cityReset = () => {
+  state.cityTextField.value = cityDefault;
+  state.headerCity.textContent = state.cityTextField.value;
+};
+
+const landscapes = {
+  coldLandscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+  coolLandscape: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+  warmLandscape: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+  hotLandscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+};
+
+const skyOptions = {
+  sunny: "☁️ ☀️ ☁️ ☀️ ☀️  ☁️",
+  cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+  rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+  snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+};
+
+const landscapeUpdate = () => {
+  let newLandscape;
+  if (state.currentTemperature >= 80) {
+    newLandscape = landscapes.hotLandscape;
+  } else if (state.currentTemperature >= 70) {
+    newLandscape = landscapes.warmLandscape;
+  } else if (state.currentTemperature >= 60) {
+    newLandscape = landscapes.coolLandscape;
+  } else {
+    newLandscape = landscapes.coldLandscape;
+  }
+
+  state.landscapeField.textContent = newLandscape;
+};
+
+//--- wave 5
+const skyOptionsUpdate = () => {
+  let skyValue = state.skyOptions.value;
+  if (skyValue === "2") {
+    state.skyField.textContent = skyOptions.sunny;
+  } else if (skyValue === "3") {
+    state.skyField.textContent = skyOptions.cloudy;
+  } else if (skyValue === "4") {
+    state.skyField.textContent = skyOptions.rainy;
+  } else if (skyValue === "5") {
+    state.skyField.textContent = skyOptions.snowy;
+  } else {
+    state.skyField.textContent = "";
+  }
+};
+
+const weatherOptions = {
```

Comment:
Nice data-driven approach for mapping from the API weather values to your selected sky values.